test/spec/uri-options/connection-options.json

```diff
@@ -179,6 +180,18 @@
         "loadBalanced": true
       }
     },
+    {
```

Comment:
which spec repo commit did this spec change come from?

Reply:
https://github.com/mongodb/specifications/commit/33916af7c70397732c203b4c3f55c7bbf8e2aebc

and 

https://github.com/mongodb/specifications/commit/cb541a15ec3dcaa91d1ba3867330b45ddb22ce93

Adding the `options: {}` seemed like a small change, and our test still passed so it seemed like a nice update.

wrt these tests - we were actually testing old behavior that was not spec compliant.  We were failing the test because we are spec compliant, so updating the test fixes the issue 😄

Reply:
These tests are now completely in sync with the specs repo too.

Reply:
Yup, thanks for getting those squared away! I just wanted to make sure we weren't keeping any related NODE tickets open and that the sync was complete if we're pulling in the changes. Crossing t's and dotting i's 😄